app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,52 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.all
+  end
+
+  def new
+    @album = Album.new
+  end
+
+  def create
+    @album = Album.new(album_params)
+    if @album.save
+      redirect_to action: 'index'
```

Comment:
didn't know this was a thing. is this the equivalent of `albums_index`? Does it actually perform a 302 redirect? Seems like it might